README.md

```diff
@@ -0,0 +1,230 @@
+
+### Tech Stack <a name="tech-stack"></a>
+> I use HTML/js  REACT for the front-end.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- *[JS OBJECTS]*
+- *[BASIC HTML]*
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[LIVE DEMO](https://develop--zesty-mooncake-1a3672.netlify.app/)
+
+<!-- - [Live Demo Link]() -->
+
+
+
+https://user-images.githubusercontent.com/68339870/227716785-9e528307-5ecf-4663-a7ad-c574599736dd.mp4
+
+
+
+https://user-images.githubusercontent.com/68339870/227717467-2454ec53-0c92-492e-b0be-3d6121e44fb8.mp4
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project :
+
+<!--
+Example command:
+sh
+ gem install rails
+
+ -->
+
+You need to have Node.js installed.
+
+### Setup
+https://github.com/amklass11/react-capsyone.git
+
+
+### Install
+
+sh
+npm install
+
+
+### Usage
+
+Run following command to start development server with live update.
+
+sh
+npm run start
+
+
+To build the app
+
+sh
+npm run build
+
+
+### Run tests
+
+There is no tests in this project but if you want to check linter errors run following command.
+
+sh
+npm run linters
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Amk Lass**
+
+
+- GitHub:[@amklass11](https://github.com/amklass11)
+
+- LinkedIn:[@AmkLAss](https://www.linkedin.com/in/amk-lass-521565196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- [ ] *[Add drag and drop function]*
+- [ ] *[Update UI/UX Design]*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please don't forget to follow me and give me advice so that I can improve more.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+-I would like to thank Microverse.
+- I wpould like to acknowledge Nelson Sakwa Abdullah for the design of this project
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [Optional][Highly recommended] It would be better if you add links to Nelson Sakwa Behance's profile and her design link also.
[hint] You can make it like that:
I want to give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).